src/Skills/Systems/BowAttackSystem.gd

```diff
@@ -7,11 +7,14 @@ func _init(event_bus):
 	_event_bus = event_bus
 	_event_bus.connect("bow_attack", self, 'activate')
 
-func activate(target: Vector2):
+func activate(source, target: Vector2):
```

Comment:
Can we type `source` as something?

Reply:
It's an entity, and using type hints would require preloading the `Entity` class... I guess a clearer variable name would do.